jwst/ami/ami_analyze_step.py

```diff
@@ -10,10 +10,8 @@ class AmiAnalyzeStep(Step):
     """
 
     spec = """
-        # Oversampling factor
-        oversample = integer(default=3,min=1)
-        # Rotation initial guess (deg)
-        rotation = float(default=0.0)
+        oversample = integer(default=3, min=1)  # Oversampling factor
+        rotation = float(default=0.0)           # Rotation initial guess [deg]
```

Comment:
That's precisely what the issue was.  Can't have parenthesis in comments.  The way it was implement worked, but it left off the comments in the ConfigObj, and `stspec` didn't print them correctly.  I've verified this works.

Reply:
Just to show an example.  Previously:
```
In [1]: from jwst.ami import AmiAnalyzeStep

In [2]: step = AmiAnalyzeStep()
2018-08-17 14:32:50,348 - stpipe.AmiAnalyzeStep - INFO - AmiAnalyzeStep instance created.

In [3]: print(step.spec)

        # Oversampling factor
        oversample = integer(default=3,min=1)
        # Rotation initial guess (deg)
        rotation = float(default=0.0)
    

In [4]: from jwst.stpipe.config_parser import load_spec_file

In [5]: load_spec_file(AmiAnalyzeStep)
Out[5]: ConfigObj({'oversample': 'integer(default=3,min=1)', 'rotation': 'float(default=0.0)'})
```
and now
```
In [1]: from jwst.ami import AmiAnalyzeStep

In [2]: step = AmiAnalyzeStep()
2018-08-17 14:28:58,651 - stpipe.AmiAnalyzeStep - INFO - AmiAnalyzeStep instance created.

In [3]: print(step.spec)

        oversample = integer(default=3, min=1)  # Oversampling factor
        rotation = float(default=0.0)           # Rotation initial guess [deg]
    
In [4]: from jwst.stpipe.config_parser import load_spec_file

In [5]: load_spec_file(AmiAnalyzeStep)
Out[5]: ConfigObj({'oversample': 'integer(default=3, min=1)  # Oversampling factor', 'rotation': 'float(default=0.0)           # Rotation initial guess [deg]'})
```
So you can see that ConfigObj now correctly picks up the comment string.  Before it did not, since it was on a separate line.